src/mimefactory.rs

```diff
@@ -169,27 +170,31 @@ impl<'a> MimeFactory<'a> {
             context
                 .sql
                 .query_map(
-                    "SELECT c.authname, c.addr  \
+                    "SELECT c.authname, c.addr, c.id \
                  FROM chats_contacts cc  \
                  LEFT JOIN contacts c ON cc.contact_id=c.id  \
                  WHERE cc.chat_id=? AND cc.contact_id>9;",
                     (msg.chat_id,),
                     |row| {
                         let authname: String = row.get(0)?;
                         let addr: String = row.get(1)?;
-                        Ok((authname, addr))
+                        let id: ContactId = row.get(2)?;
+                        Ok((authname, addr, id))
                     },
                     |rows| {
                         for row in rows {
-                            let (authname, addr) = row?;
+                            let (authname, addr, id) = row?;
                             if !recipients_contain_addr(&recipients, &addr) {
                                 recipients.push((authname, addr));
                             }
+                            recipient_ids.insert(id);
                         }
                         Ok(())
                     },
                 )
                 .await?;
+            let recipient_ids: Vec<_> = recipient_ids.into_iter().collect();
+            ContactId::scaleup_origin(context, &recipient_ids, Origin::OutgoingTo).await?;
```

Comment:
The documentation says that for contacts that we have already sent messages to, it must be >= `OutgoingTo`, also this is needed to reject authname updates for such contacts while allowing updates if we import several vCards in a row